statelesscni/internal.go

```diff
@@ -0,0 +1,46 @@
+// Copyright 2017 Microsoft. All rights reserved.
+// MIT License
+
+package statelesscni
+
+import (
+	"encoding/json"
+
+	"github.com/Azure/azure-container-networking/statelesscni/util"
+	cniSkel "github.com/containernetworking/cni/pkg/skel"
+	cniTypes "github.com/containernetworking/cni/pkg/types"
+)
+
+const (
+	Internal = "internal"
+)
+
+// CallPlugin calls the given CNI plugin through the internal interface.
+func CallPlugin(plugin PluginApi, cmd string, args *cniSkel.CmdArgs, nwCfg *NetworkConfig) (*cniTypes.Result, error) {
```

Comment:
@tamilmani1989 I am not sure how this file is used ? How is this function called ?